### ISSUE TITLE
Choose which way to Sign up setting

### ISSUE THREAD

**Issue by antho2305** [humhub/humhub#5753]
### Is your feature request related to a problem? Please describe.
It would be great to let admin choose which way users can sign up
### Describe the solution you'd like
We can choose between direct form (without the need to use mail with link etc...) and access directly to register form
### Describe alternatives you've considered
Nowadays everyone can create an email adress I might be better to got a register button an access to the form and why not ask for a OPT (one time password) for security, but let admin choose which is better. For example by my side the system I build is not useful using the mail system.

**Comment by luke-** [comment]
Basically, this is about not forcing an email address?

https://github.com/humhub/humhub/issues/1381
https://github.com/humhub/humhub/issues/3191

**Comment by antho2305** [comment]
@luke- no, just the possibility to sign up directly without the need to use email link.

ex. The actual system is when I'm in the login/sign up screen : Sign up/register -> Enter mail and capcha -> It sends a mail with a link and a token-> You have to go to your webmail click on the link and you are redirected to the register form -> wait for admin validation (if set up).

It would be great that user can in the Sign up/resiter -> Enter mail and capcha -> Access directly to the resgister form and complete profil -> wait for admin validation (if set up)

If you have an idea how I can do that it would be really really great.

**Comment by luke-** [comment]
@antho2305 When does the user need to confirm their email address in your workflow? If no admin approval is required, can they log in directly or do they still have to click in WebMail?
(If no verified e-mail is needed, it matches the already existing issues).

**Comment by antho2305** [comment]
@luke- whatever I do users still have to use email to access to registration form. I don't know if I'm clear in my request. When a new user want to register, he has to fill in the mail fied and click register, then a "thank you page" appears and ask the user to go in its webmail to click on the link to complete its registration... Then when the user click on the link the mail, he is redirected on the register form. What I would like is that the user fill in the mail fied, click "register" then access directly to the registration form. See screens bellow

**Actual system :**
![screen2](https://user-images.githubusercontent.com/40645459/173068965-55b96749-b418-4cad-b6b4-1a17d37ed29e.png)
![screen3](https://user-images.githubusercontent.com/40645459/173068974-0d943488-9f74-4901-bcdb-b719d6a66abf.png)

What I would like **Direct registration :** 

![screen4](https://user-images.githubusercontent.com/40645459/173069918-9e6db5d8-0c64-40f7-afd3-75de73b3446e.png)

**Comment by luke-** [comment]
I understood that. As soon as the issues or feature requests with the optional email address are implemented, it could work like this.

**Comment by ralfblask** [comment]
If you do so, everybody can write a bot that registers thousands of users per second, good luck
